core/src/test/resources/Validation/Basic/xsd/A_correct.xsd-after-adding-namespace.xsd

```diff
@@ -3,7 +3,7 @@
            targetNamespace="http://www.ing.com/testxmlns" xmlns:a="http://www.ing.com/testxmlns"
            xmlns:b="http://xmlns/b"  xmlns="http://www.ing.com/testxmlns"
     >
-  <xs:import namespace="http://xmlns/b" schemaLocation="B.xsd" />
+  <xs:import namespace="http://xmlns/b"/>
```

Comment:
Removed, because the production code strips the schema location but the previous test-only method didn't.